core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/DCAT.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.

```

Comment:
Very minor this, but this is not the correct copyright header for new contributions. The correct header is as follows:

`/**
 * Copyright (c) 2015 Eclipse RDF4J contributors, and others.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Distribution License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/org/documents/edl-v10.php.
 */`

(basically the mention of "Aduna" shouldn't be there for new code).